influxdb_store.go

```diff
@@ -145,6 +173,85 @@ func (in *InfluxDBStore) Trace(id ID) (*Trace, error) {
 	return trace, nil
 }
 
+func mustJSONFloat64(x interface{}) float64 {
+	n := x.(json.Number)
+	v, err := n.Float64()
+	if err != nil {
+		panic(err)
+	}
+	return v
+}
+
+func mustJSONInt64(x interface{}) int64 {
+	n := x.(json.Number)
+	v, err := n.Int64()
+	if err != nil {
+		panic(err)
+	}
+	return v
+}
+
+// Aggregate implements the Aggregator interface.
+func (in *InfluxDBStore) Aggregate(start, end time.Duration) ([]*AggregatedResult, error) {
+	// Find the mean (average), minimum, maximum, std. deviation, and count of
+	// all spans.
+	q := `SELECT MEAN("duration"),MIN("duration"),MAX("duration"),STDDEV("duration"),COUNT("duration") from spans`
```

Comment:
I appreciate the notes on continuous queries! I think I originally saw these but didn't understand exactly what they did, but your explanation makes it much more clear. Certainly a good idea to investigate in a follow-up PR.